Pedro.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace EjerciciooUno
+{
+    internal class Pedro:Persona
+
+
+    {
+        // se crea metodo para pedro camianr
+        public void caminar ()
```

Comment:
Delete the space between the name of the method and the parenthesis 

`public void caminar()`

Reply:
has been corrected